infra/base-images/base-clang/checkout_build_install_llvm.sh

```diff
@@ -93,6 +98,24 @@ fi
 git -C $LLVM_SRC checkout $LLVM_REVISION
 echo "Using LLVM revision: $LLVM_REVISION"
 
+if [ -n "$INTROSPECTOR_PATCHES" ]; then
+  # For fuzz introspector.
+  echo "Applying introspector changes"
+  BBBASE=$PWD
```

Comment:
Maybe call this variable OLD_WORKING_DIR instead of `BBBASE`

Reply:
addressed in  #7199